rules/sonic-platform-common.mk

```diff
@@ -3,6 +3,7 @@
 SONIC_PLATFORM_COMMON_PY2 = sonic_platform_common-1.0-py2-none-any.whl
 $(SONIC_PLATFORM_COMMON_PY2)_SRC_PATH = $(SRC_PATH)/sonic-platform-common
 $(SONIC_PLATFORM_COMMON_PY2)_PYTHON_VERSION = 2
+$(SONIC_PLATFORM_COMMON_PY2)_DEPENDS = $(SONIC_CONFIG_ENGINE)
```

Comment:
Is this dependency needed at build time? If not, a runtime dependency should suffice (`$(SONIC_PLATFORM_COMMON_PY2)_RDEPENDS`). Can you determine whether this will be sufficient?

Reply:
Sure, @jleveque . let me check.

Reply:
Yes, it has a build-time dependency.